### DISCUSSION TITLE
build failed

### DISCUSSION THREAD

**Discussion by dankorea** [shuup/shuup#2505] [Q&A]
#6 413.4  unable to fsync updated status of 'python3-pip': Input/output error
#6 416.3 E: Sub-process /usr/bin/dpkg returned an error code (2)
------
executor failed running [/bin/sh -c apt-get update     && apt-get --assume-yes install         libpangocairo-1.0-0         python3         python3-dev         python3-pil         python3-pip     && rm -rf /var/lib/apt/lists/ /var/cache/apt/]: exit code: 100
ERROR: Service 'shuup' failed to build : Build failed

anyone could tell me what's the problem?

**Comment by OlehObodin** [comment]
Same problem
